src/index.prod.html

```diff
@@ -0,0 +1,34 @@
+<!doctype html>
```

Comment:
Can we somehow preserve the matching between this file and index.html and maybe the webpack files?

Reply:
What kind of matching are we looking for here?

Reply:
The files have many similar lines, we don't want people to just change one of the files but always all.

Reply:
Added a comment to four files, index, index.prod, header and footer js files to contact you or me before modifying it.

Reply:
Please describe clearly in these comments what constraint folks need to follow and why. This will also help reviewers who are looking at these files, especially in cases where both you and @vojtechjelinek might not be around.

Reply:
Done!